.github/workflows/alpha-release.yml

```diff
@@ -84,8 +84,7 @@ jobs:
 
       # Clean package.json
       - name: Remove scripts, devDependencies, and beachball from package.json
-        run: node ./common/scripts/preparePackageJSON.js
-        working-directory: ./packages/communication-react
```

Comment:
Without this, how does it know to change the package.json in communication-react?

![image](https://github.com/Azure/communication-ui-library/assets/2684369/dbf3d196-d4f0-4765-91bc-90be607ed939)

Reply:
ill need to look further into this

Reply:
Should be fixed now